### PR TITLE
Allow the Bodhi team to ack for Mergify for my PRs.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
I've reported the Mergify syntax error here: https://github.com/Mergifyio/mergify-engine/issues/310

File: .mergify.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ pull_request_rules:
   conditions:
   - label!=no-mergify
   - author=bowlofeggs
-  - approved-reviews-by=@fedora-infra
+  - "#approved-reviews-by>=1"
```
Comment (Zlopez) [inline .mergify.yml:41]: Why did you commented this and added >=1 at the same time?

**Comment by bowlofeggs** [comment]
On Thu, 2018-12-06 at 01:09 -0800, Zlopez wrote:
> Why did you commented this and added >=1 at the same time?

It's not a comment, it's the syntax for indicating a number:

https://doc.mergify.io/conditions.html#matching-pull-requests-passing-travis-and-being-approved

There is a bug in mergify that prevents it from matching the Bodhi
team, which is why I am making this change.

**Review by Zlopez — APPROVED** [review]

**Comment by Zlopez** [comment]
I looked to the mergify documentation, but probably on the wrong place. In this case LGTM.

**Pull request merged**